index.js

```diff
@@ -4,10 +4,15 @@ const DefinePlugin = require('webpack').DefinePlugin;
 
 module.exports = DotenvPlugin;
 
-function DotenvPlugin(options) {
-  dotenv.config(options || {});
+const defaultOptions = {
+  sample: './.env.default',
+  path: './.env'
+};
+
+function DotenvPlugin(options = defaultOptions) {
```

Comment:
I just realized ES6 default params didn't ship until node v6, so this will require at least that version, or a transpiler, to work.

If you prefer, I could change this back and do a `options = options || defaultOptions` inside the constructor.